`npm run typecheck` fails on fresh project

### Have you experienced this bug with the latest version of the template?

yes

### Steps to Reproduce

Create a bluestack via  `npx create-remix@latest` and choose bluestack. Follow instructions, run `npm run typecheck`.

### Expected Behavior

Tests should pass out of the box.

### Actual Behavior

Tests fail:

```shell
npm run typecheck 

> typecheck
> tsc && tsc -p cypress

error TS2688: Cannot find type definition file for '@testing-library/cypress'.
  The file is in the program because:
    Entry point of type library '@testing-library/cypress' specified in compilerOptions

  cypress/tsconfig.json:15:34
    15     "types": ["node", "cypress", "@testing-library/cypress"],
                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~
    File is entry point of type library specified here.

error TS2688: Cannot find type definition file for 'cypress'.
  The file is in the program because:
    Entry point of type library 'cypress' specified in compilerOptions

  cypress/tsconfig.json:15:23
    15     "types": ["node", "cypress", "@testing-library/cypress"],
                             ~~~~~~~~~
    File is entry point of type library specified here.


Found 2 errors.
```

--- a/cypress/tsconfig.json
+++ b/cypress/tsconfig.json
@@ -20,7 +20,6 @@
     "strict": true,
     "skipLibCheck": true,
     "resolveJsonModule": true,
-    "typeRoots": ["../types", "../node_modules/@types"],
 
     "paths": {
       "~/*": ["../app/*"]

